### PR TITLE
CRYPTO-5: Create site for apache commons crypto

### PR DESCRIPTION
**Pull request by JkSelf**
This fix https://issues.apache.org/jira/browse/CRYPTO-5

### CODE REVIEW

File: src/site/site.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <href>/index.html</href>
+        <alt>Commons Crypto&trade; logo</alt>
+    </bannerRight>
+    <body>
+        <menu name="Documentation">
+            <item name="Overview"      href="/index.html"/>
+            <!-- Should use download_crypto.cgi at apache host -->
+            <!-- <item name="Download"
+                   href="http://commons.apache.org/crypto/download_crypto.cgi"/> -->
+            <item name="Download" href="/download_crypto.html"/>
```
Comment (sundapeng) [inline src/site/site.xml]: We can use http://commons.apache.org/crypto/download_crypto.cgi now.

File: src/site/xdoc/proposal.xml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+   <li>
+       Confluence FAQ - New category ¡°commons-crypto" (when available).
+   </li>
+  </ul>
+
+  </subsection>
+<subsection name="(4) Initial Committers">
+
+  <p>
+   The initial committers on the Chimera component shall be as follows:
```
Comment (jerrychenhf) [inline src/site/xdoc/proposal.xml]: Dapeng, please change this "Chimera" to Apache Commons Crypto.
Keep the below "Chimera" instances unchanged.

Comment (JkSelf) [inline src/site/xdoc/proposal.xml]: Thanks @jerrychenhf @sundapeng comments. I have fixed.

File: src/site/xdoc/index.xml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        the minimum code and effort. Please note that Apache Commons Crypto doesn't implement the
+        cryptographic algorithm such as AES directly. It wraps to Openssl or JCE which implement the
+        algorithms.
+    </p>
+</section>
+<!-- ================================================== -->
+<section name="Features">
+  <p>
+    <ul>
+      <li>Cipher API for low level cryptographic operations.</li>
```
Comment (jerrychenhf) [inline src/site/xdoc/index.xml]: Please help add the following after Cipher API item:
<li>Secure true random number generator.<li>

File: src/site/xdoc/index.xml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+      <li>Java stream API (CryptoInputStream/CryptoOutputStream) for high level stream
```
Comment (jerrychenhf) [inline src/site/xdoc/index.xml]: Remove "(CryptoInputStream/CryptoOutputStream)" as we now have several stream implementations.

File: src/site/xdoc/index.xml [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+        encyrption/decryption.
+      </li>
+      <li>Both optimized with high performance AES encryption/decryption. (1400 MB/s - 1700 MB/s
```
Comment (jerrychenhf) [inline src/site/xdoc/index.xml]: `Both optimized with high performance AES encryption/decryption. (1400 MB/s - 1700 MB throughput in modern Xeon processors).
`
Change to: 
`High performance AES encryption/decryption optimized with Intel AES-NI.`

File: src/site/xdoc/index.xml [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+        throughput in modern Xeon processors).
+      </li>
+      <li>JNI-based implementation to achieve comparable performance to the native C++ version
```
Comment (jerrychenhf) [inline src/site/xdoc/index.xml]: Remove this item:
`JNI-based implementation to achieve comparable performance to the native C++ version based on Openssl. 
`

File: src/site/xdoc/index.xml [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+        throughput in modern Xeon processors).
+      </li>
+      <li>JNI-based implementation to achieve comparable performance to the native C++ version 
+          based on Openssl.
+      </li>
+      <li>Portable across various operating systems (currently only Linux); Apache Commons Crypto loads the
+          library according to your machine environment (It looks system properties, os.name and
+          os.arch).
+      </li>
+      <li>Simple usage. Add the crypto-(version).jar file to your classpath.</li>
```
Comment (jerrychenhf) [inline src/site/xdoc/index.xml]: Should be commons-crypto-(version).jar?

File: src/site/xdoc/mail-lists.xml [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+
+    <section name="Overview">
+      <p>
+        <a href="index.html">Apache Commons Crypto</a> shares mailing lists with all the other
+        <a href="http://commons.apache.org/components.html">Commons Components</a>.
+        To make it easier for people to only read messages related to components they are interested in,
+        the convention in Commons is to prefix the subject line of messages with the component's name,
+        for example:
+        <ul>
+          <li>[crypto] Problem with the ...</li>
```
Comment (winningsix) [inline src/site/xdoc/mail-lists.xml]: Upper case [CRYPTO]

File: src/site/xdoc/index.xml (hunk 7 above)
Comment (JkSelf) [inline src/site/xdoc/index.xml]: Yes, I have changed it. Thank you for your comment.

**Comment by JkSelf** [comment]
Thanks @jerrychenhf  comments. I have fixed.

**Comment by jerrychenhf** [comment]
+1

**Comment by winningsix** [comment]
Committed to the upstream. Thanks @JkSelf for your contribution and @jerrychenhf for the reviews.

**Comment by winningsix** [comment]
@JkSelf Could you close this PR since it's committed already.

**Pull request closed without merging**